app/controllers/users_controller.rb

```diff
@@ -1,6 +1,13 @@
 class UsersController < ApplicationController
+  rescue_from JsonApiClient::Errors::ClientError, with: :render_providers
+  rescue_from JsonApiClient::Errors::ServerError, with: :render_providers
```

Comment:
I feel like this is too wide a net being cast here, I don't think we know that redirecting to providers is always going to be the right thing.

I think this should be a rescue section in `accept_transition_info` that logs the Sentry error in addition to just moving them along. Also, I worry a little bit about a circular redirect here in case the problem _isn't_ that the user has already accepted the terms, is there any way to make this more specific? Do we log an error message/condition we could use?

Reply:
Done